020_app_L44/020_app_L44_unsafe.py

```
@@ -0,0 +1,108 @@
+@app.route("/xss/<string:payload>")
+def definite_xss(payload: str) -> None:
+    content = flask.Markup(payload)
+    return render_template(content)
+
+
+@app.route("/sql/<string:payload>")
+def definite_sql(payload: str) -> None:
+# {/fact}
+    con = sqlite3.connect()
+    cur = con.cursor()
+    cur.execute(f"SELECT info FROM users WHERE name={payload}")
```

Ruff (0.14.1)

Possible SQL injection vector through string-based query construction

(S608)